eng/pipelines/coreclr/perf.yml

```diff
@@ -23,11 +23,12 @@ variables:
   - template: /eng/pipelines/common/variables.yml
 
 schedules:
-- cron: "30 2 * * *"
-  displayName: Every night at 2:30AM
+- cron: "30 2,14 * * *"
```

Comment:
This doesn't appear to be the case in main. Are we changing this in main too? or just for release/6.0?

Reply:
For now, the change is only going to be in release/6.0 as the main branch is triggered more often than that by the general runtime updates. The other reason we will push this to main is if it needs to be a trigger in main to also work in release/6.0.